### DISCUSSION TITLE
updateMany with limit ?

### DISCUSSION THREAD

**Discussion by damitai** [prisma/prisma#8342] [Q&A] [answered]
Is there any way to do `updateMany()` with LIMIT ?
would be equivalent to `UPDATE table WHERE ... LIMIT ...` SQL

**Comment by ryands17** [comment, marked as answer]
In this case, you need to fetch the values first with the limit and then perform an `updateMany` using `in` in `where` passing the id's.

**Reply by damitai** [reply]
Yup, that can be a workaround... although 2 queries instead of 1 ...
Any idea if this is planned as a feature?

**Reply by ryands17** [reply]
Not yet, but I see you've opened a feature request.

**Reply by cwqt** [reply]
Great way to give yourself a double booking problem - been over a year on this, any update?

**Reply by saesbuba** [reply]
any updates on this feature?

**Reply by ludralph** [reply]
@saesbuba No updates yet but we do have it in our radar. See the issue [here](https://github.com/prisma/prisma/issues/8359)

**Comment by steven-tey** [comment]
Looks like this is supported in `6.3.0` onwards: https://www.prisma.io/docs/orm/reference/prisma-client-reference#updatemany
